packages/web-react/src/components/Toast/useToastStyleProps.ts

```diff
@@ -0,0 +1,40 @@
+import classNames from 'classnames';
+import { useClassNamePrefix } from '../../hooks';
+import { SpiritToastProps } from '../../types';
+
+export interface ToastStyles<T> {
+  classProps: {
+    root: string;
+    queue: string;
+  };
+  props: T;
+}
+
+export function useToastStyleProps(props: SpiritToastProps): ToastStyles<SpiritToastProps> {
+  const { alignmentX, alignmentY, ...restProps } = props;
+
+  const toastClass = useClassNamePrefix('Toast');
+
+  function processAlignment(alignment: SpiritToastProps['alignmentX'] | SpiritToastProps['alignmentY']) {
+    return typeof alignment === 'object' && alignment !== null
```

Comment:
```ts
return alignments.map(alignment => typeof alignment === 'object' && alignment !== null ? ...)
```
I think you can simply iterate over both alignments and map them. This will return a new array with the accumulator from reduce.

So there is no need to call the process function twice. This is just a little optimization of how the code can look like. The decision is up to you if you implement it like this. :-)